test/skimNtuple2016_HHbtag.cpp

```diff
@@ -4453,12 +4492,8 @@ int main (int argc, char** argv)
             theSmallTree.m_bjet2_jecUnc = theBigTree.jets_jecUnc->at(iJet);
             continue ;
           }
-          TLorentzVector tlv_dummyJet(
-                                      theBigTree.jets_px->at (iJet),
-                                      theBigTree.jets_py->at (iJet),
-                                      theBigTree.jets_pz->at (iJet),
-                                      theBigTree.jets_e->at (iJet)
-                                     );
+          TLorentzVector tlv_dummyJet(theBigTree.jets_px->at(iJet), theBigTree.jets_py->at(iJet), theBigTree.jets_pz->at(iJet), theBigTree.jets_e->at(iJet));
+          if (doSmearing) tlv_dummyJet = Smearer.getSmearedJet(tlv_dummyJet, theBigTree);
```

Comment:
Please double check if this is the only point where we should shift other jets. To me yes because everything is done inside this loop.

Reply:
Yes, they are also smearead in line 4960 where we re-define them to store the HHBtag score.